### PR TITLE
Update brach to latest version of the action

### PR DESCRIPTION
**Pull request by hernandoagf**

### CODE REVIEW

File: src/parseGithubMetadata.ts [hunk 1]
```diff
@@ -1,69 +1,89 @@
...
+        : oldVoteTypeToNewParameters(voteType)
+
+      if (errorParameters.length > 0 || !parameters) {
+        throw new Error(
+          `Invalid poll parameters for poll ${poll.pollId}. ${errorParameters}`
+        )
+      }
+
+      let startDate, endDate
+      //poll coming from poll create page
```
Comment (tyler17) [inline src/parseGithubMetadata.ts]: why is this necessary? Are you saying polls coming from poll create page don't have the start and end date set posted on-chain?

**Review by tyler17 — APPROVED** [review]
Just had one question, but otherwise looks good. I forgot, how did we decide to make sure this action gets run periodically?

File: src/parseGithubMetadata.ts (hunk 1 above)
Comment (hernandoagf) [reply src/parseGithubMetadata.ts]: Actually, I recycled that part from the parsing metadata function from the Gov Portal, which can be called directly from the create page, passing a partial poll with 0 startDate and endDate. But in this GitHub action, that code should never be reachable as polls coming from Spock should always have start and end dates.

**Comment by hernandoagf** [comment]
> Just had one question, but otherwise looks good. I forgot, how did we decide to make sure this action gets run periodically?

@tyler17 by using [these triggers](https://github.com/hernandoagf/community/blob/master/.github/workflows/aggregate-polls.yml#L2-L15) we can ensure that the action runs:
1. Automatically, every time a poll metadata file changes
2. Manually, when someone with the right permissions (GovAlpha) triggers the action
3. On a schedule, setting up a cron job

Then, if any of the poll files changed, it will open a PR with the new aggregated file and the new hash file, if no files changed, the action will just skip the PR and complete silently

File: src/parseGithubMetadata.ts (hunk 1 above)
Comment (tyler17) [reply src/parseGithubMetadata.ts]: Got it, we could remove the code or update the comment then just for readability

Comment (hernandoagf) [reply src/parseGithubMetadata.ts]: Done, removed the code to always use start and end dates from Spock, gave a quick check to the aggregated file and it looks good, this is [the file](https://raw.githubusercontent.com/hernandoagf/community/master/governance/polls/meta/polls.json) in case you want to check it out
